lib/commons/dom/elements-from-point.js

```diff
@@ -40,15 +47,68 @@ dom.elementsFromPoint = function (doc, x, y) {
 
 		// add "pointer-events: none", to get to the underlying element
 		current.style.setProperty('pointer-events', 'none', 'important');
-
-		if (dom.isOpaque(current)) { break; }
 	}
 
 	// restore the previous pointer-events values
 	for (i = previousPointerEvents.length; !!(d = previousPointerEvents[--i]);) {
 		elements[i].style.setProperty('pointer-events', d.value ? d.value : '', d.priority);
 	}
 
+	elements = dom.reduceToElementsBelowFloating(elements, targetNode);
+
 	// return our results
-	return elements;
+	return dom.reduceToFirstOpaque(elements);
+};
+
+/**
+ * Reduce an array of elements to only those that are below a 'floating' element.
+ * 
+ * @param {Array} elements
+ * @param {Element} targetNode
+ * @returns {Array}
+ */
+dom.reduceToElementsBelowFloating = function (elements, targetNode) {
+	var floatingPositions = ['fixed', 'sticky'], finalElements = [], targetFound = false, index, currentNode, style;
+	
+	// Filter out elements that are temporarily floating above the target
+	for (index = 0; index < elements.length; ++index) {
+		currentNode = elements[index];
+		if (currentNode === targetNode) {
+			targetFound = true;
+		}
+
+		style = window.getComputedStyle(currentNode);
+
+		if (!targetFound && floatingPositions.indexOf(style.position) !== -1) {
+			//Target was not found yet, so it must be under this floating thing (and will not always be under it)
+			finalElements = [];
+			continue;
+		}
+
+		finalElements.push(currentNode);
+	}
+	
+	return finalElements;
+};
+
+/**
+ * Reduce an array of elements to only those before the first opaque
+ * 
+ * @param {Array} elements array of elements to reduce
+ * @returns {Array}
+ */
+dom.reduceToFirstOpaque = function (elements) {
+	var finalElements = [], currentNode, index;
```

Comment:
fixed in https://github.com/dequelabs/axe-core/pull/205/commits/23070e79f5ca57d8d71514ea7a7b673cd89308ba